pkg/abstractions/endpoint/buffer.go

```diff
@@ -434,6 +434,9 @@ func (rb *RequestBuffer) handleHttpRequest(req *request, c container) {
 	}
 
 	containerUrl := fmt.Sprintf("http://%s/%s", c.address, req.ctx.Param("subPath"))
+	if req.task.msg.NoOp {
```

Comment:
Feels like there a simpler way to do this that doesn't "trick" the system into warming up the task queue. Why not just have the warm up endpoint launch a container and set a keep warm on it so the autoscaler can't shut it down?